os/arch/arm/src/armv7-m/up_lazyexception.S

```diff
@@ -245,8 +245,12 @@ exception_common:
 	 * here prohibits nested interrupts without some additional logic!
 	 */
 
-	setintstack	r2, r3
-
+	ldr		r5, =g_nestlevel
+	ldr		r6, [r5]
```

Comment:
Can you leave a comment over here, "If g_nestlevel is zero then behave as normal, switching from the user to the interrupt stack; if g_nestlevel is greater than zero, then do not switch stacks. In this latter case, we are already using the interrupt stack."

Reply:
Done